vignettes/social_gain_vignette/readme.md

```diff
@@ -14,25 +14,25 @@ MPOG calculates performance benchmarks and averages for each institution. One co
 ### Healthcare professional performance
 Chikondi, a resident physician at Max Community Hospital, has the following performance data over the last 6 months for the OPIOID measure:
 
-|Month|Performance Level|Top 10% Benchmark|Peer Average|
-|-----|-----------------|-----------------|------------|
-|Oct  |              84%|               94|          85|
-|Nov  |              85%|               95|          87|
-|Dec  |              85%|               93|          85|
-|Jan  |              87%|               96|          88|
-|Feb  |              88%|               97|          87|
-|Mar  |              91%|               95|          86|
+|Month|Performance Level|Peer Average|75th Percentile Benchmark|90th Percentile Benchmark|
+|-----|-----------------|------------|-------------------------|-------------------------|
+|Jul  |              85%|          85|                       88|                       92|
+|Aug  |              85%|          85|                       88|                       92|
+|Sept |              85%|          85|                       88|                       92|
+|Oct  |              85%|          85|                       88|                       92|
+|Nov  |          **85%**|          85|                       88|                       92|
+|Dec  |          **94%**|          85|                       88|                       92|
 
 Fahad, an attending pediatric anesthesiologist at Max Community Hospital, has the following performance data over the last 6 months for the OPIOID measure:
 
-|Month|Performance Level|Top 10% Benchmark|Peer Average|
-|-----|-----------------|-----------------|------------|
-|Oct  |              95%|               94|          85|
-|Nov  |              95%|               95|          87|
-|Dec  |              94%|               93|          85|
-|Jan  |              92%|               96|          88|
-|Feb  |              94%|               97|          87|
-|Mar  |              97%|               95|          86|
+|Month|Performance Level|Peer Average|75th Percentile Benchmark|90th Percentile Benchmark|
+|-----|-----------------|------------|-------------------------|-------------------------|
+|Jul  |              90%|          90|                       92|                       96|
+|Aug  |              90%|          90|                       92|                       96|
+|Sept |              90%|          90|                       92|                       96|
+|Oct  |              90%|          90|                       92|                       96|
+|Nov  |          **93%**|          90|                       92|                       96|
+|Dec  |          **97%**|          90|                       92|                       96|
```

Comment:
Comparators:
- [x] 90th percentile
- [ ] 75th percentile
- question, we want preferences to determine that 75th percentile social gain isn't going to be an accepted message candidate, correct?